firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -117,6 +121,13 @@ final class HomepageViewController: UIViewController,
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
         wallpaperView.updateImageForOrientationChange()
+        store.dispatch(
+            TopSitesAction(
+                numberOfTilesPerRow: numberOfTilesPerRow(for: size.width),
+                windowUUID: self.windowUUID,
+                actionType: TopSitesActionType.updatedNumberOfTilesPerRow
+            )
+        )
```

Comment:
was trying to test this, but would need to mock `UIViewControllerTransitionCoordinator` which comes with a bunch of methods, we can UI tests instead to this out as well